app/Resources/assets/scripts/components/numbers.js

```diff
@@ -72,18 +72,36 @@
 
                 return (
                     <div className="numbers">
-                        <Editable className={estimateClass} title="Estimate" handleInput={handleEstimateInput}>{estimate}</Editable>
-                        <Editable className={consumedClass} title="Consumed" handleInput={handleConsumedInput}>{consumed}</Editable>
-                        <Editable className={remainingClass} title="Remaining" handleInput={handleRemainingInput}>{remaining}</Editable>
+                        <div className={estimateClass}>
+                            <label>Estimé</label>
```

Comment:
L'utilisation de `label` est incorrecte ici : il n'est associé à aucun élément de formulaire (que ce soit par l'attribut `for` ou en l'englobant directement). `contentEditable` != `input` :)